### PR TITLE
[Bug 14956] lc-compile: Remove warning for `metadata` before `use`.

### PR DESCRIPTION
**Pull request by peter-b**

### CODE REVIEW

File: toolchain/lc-compile/src/grammar.g [hunk 1]
```diff
@@ -212,7 +212,6 @@
         Metadata(-> Left) Separator
         PreImportMetadata(-> Right)
         where(Left -> metadata(Position, _, _))
```
Comment (runrevmark) [inline toolchain/lc-compile/src/grammar.g]: You can remove this line too - it is fetching the position to use in the warning.

File: docs/lcb/notes/14956.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# LiveCode Builder Language
+## Syntax
+
+* `metadata` definitionts may now occur anywhere a module's top-level
```
Comment (runrevmark) [inline docs/lcb/notes/14956.md]: Typo: definitionts should be definitions.

**Comment by runrevmark** [comment]
Can you check the builder reference too?

**Comment by peter-b** [comment]
@runrevmark I think these updates address your comments -- thanks!

**Comment by runrevmark** [comment]
Reviewed-by: @runrevmark.

**Pull request merged**